scripts/check_gofmt.sh

```diff
@@ -4,5 +4,5 @@
 # Fail if a .go file hasn't been formatted with gofmt
 set -euo pipefail
 
-GO_FILES=$(find . -iname '*.go' -type f | grep -v /vendor/)   # All the .go files, excluding vendor/
+GO_FILES=$(find . -iname '*.go' -type f)   # All the .go files
 test -z $(gofmt -s -d $GO_FILES | tee /dev/stderr)
```

Comment:
Wrap double quotes around variables and command substitution, so `"$GOFILES"` and it should stop complaining